EntAppProject/src/main/java/com/jobook/EntAppProject/model/Address.java

```diff
@@ -1,8 +1,9 @@
 package com.jobook.EntAppProject.model;
 
-import org.springframework.stereotype.Component;
+import javax.validation.constraints.NotNull;
 
 public class Address {
+	@NotNull

```

Comment:
Consider doing this in your DTOs. Then, when you have your endpoints, or just in general, when the address DTO gets constructed, you can do `@Valid Address`. I didn't apply it to the rest of your DTOs, but just consider this.